chapters/fa/chapter2/2.mdx

```diff
@@ -405,9 +405,9 @@ tf.Tensor(
 </div>
 
 
-حالا می‌توانیم ببینیم که پیش‌بینی مدل برای جمله اول `[0.9995,  0.0005]` و برای جمله دوم `[0.9995,  0.0005]` است. این‌ها مقادیر آشنای آماری هستند.
+حالا می‌ببینیم که پیش‌بینی مدل برای جمله اول `[0.9995,  0.0005]` و برای جمله دوم `[0.9995,  0.0005]` است. این‌ها مقادیر آشنای آماری هستند.
```

Comment:
- Quick fix: values are the same for first and second sentence, first sentence output values should be replaced respectively.
- Same point about using `<span>` tag with a ltr direction attribute for outputs.

Suggestion:
"این‌ها مقادیر آشنای آماری (به فرم احتمال) هستند."
instead of:
"این‌ها مقادیر آشنای آماری هستند."

Reply:
Applied all.